### PR TITLE
update: remove implicit react deps and prompt for install

### PR DESCRIPTION
**Pull request by eanplatter**
Per discussion here: https://github.com/eanplatter/enclave/issues/17

### CODE REVIEW

File: src/postinstall/prompts.js [hunk 1]
```diff
@@ -12,36 +12,36 @@ var prompts = [
...
     required: false,
-    name: 'port'
+    name: 'port',
   },
   {
-    description: chalk.cyan('Where does (or will) your index.html file live?'),
-    default: 'index.html',
+    description: chalk.red('Would you like me to install React for you?'),
+    default: 'no',
     type: 'string',
```
Comment (FiberJW) [inline src/postinstall/prompts.js]: Why isn't the type a boolean?

Comment (FiberJW) [inline src/postinstall/prompts.js]: Make it easier for yourself. Using a string will just make for horrible case testing.

Comment (eanplatter) [inline src/postinstall/prompts.js]: haha geez idk what I was thinking. Thanks :D

**Comment by FiberJW** [comment]
LGTM

**Comment by eanplatter** [comment]
LGTM

**Pull request merged**